game-store-service/functions/getGameById.ts

```diff
@@ -1,44 +1,45 @@
 import { APIGatewayProxyHandler } from "aws-lambda";
-import { readFileSync } from "fs";
-import * as path from "path";
-import { GameList, Game } from "../types";
-import { delay } from "../delay";
+import { invokeClient, closeClient } from './invokeClient';
 
 const headers = {
   "Access-Control-Allow-Origin": "*", // Required for CORS support to work
   "Access-Control-Allow-Credentials": true, // Required for cookies, authorization headers with HTTPS
 };
 
-const gamesList: GameList = JSON.parse(
-  readFileSync(path.join(__dirname, "data", "games.json"), "utf-8")
-);
-
-const getGameById: APIGatewayProxyHandler = async (event, _context) => {
+const getGameById: APIGatewayProxyHandler = async (_event, _context) => {
+  let client;
   try {
     const {
-      pathParameters: { id },
-    } = event;
+      pathParameters: { id: gameId },
+    } = _event;
+    console.debug('Get product by id request with params: \n', JSON.stringify(_event.pathParameters,null, 2));
 
-    const game = await delay<Game>(
-      () => gamesList.find((game) => game.id === id),
-      150
+    client = await invokeClient();
+    const {rows: [foundItem]} = await client.query(
+      'select p.*, s.count from products p join stocks s on s.product_id = p.id where p.id = $1',
+      [gameId]
```

Comment:
I suggest you should check whether the gameId is matches UUID format to prevent HTTP 500 errors in requests like https://x5v02w7ua0.execute-api.eu-west-1.amazonaws.com/dev/products/12